File: id-card-toolkit-ios-sdk-v3.1.6/id-card-toolkit-ios-sdk-v3.1.6/samples/IDCardToolkitObjCApp/IDCardToolkitObjCApp/CustomTableViewCell/CustomTableViewCell.m
//
//  CustomTableViewCell.m
//  
//
//  Created by Federal Authority For Identity and Citizenship  on 1/9/17.
//  Copyright © 2017 Federal Authority For Identity and Citizenship. All rights reserved.
//

#import "CustomTableViewCell.h"

@implementation CustomTableViewCell
@synthesize keyData,valueData;

- (void)awakeFromNib {
    [super awakeFromNib];
    
    keyData.adjustsFontSizeToFitWidth=YES;
    valueData.adjustsFontSizeToFitWidth=YES;
    
    // Initialization code
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

@end

File: id-card-toolkit-ios-sdk-v3.1.6/id-card-toolkit-ios-sdk-v3.1.6/samples/ICASwiftApp/ICASwiftApp/CustomTableViewCell.swift
//
//  CustomTableViewCell.swift
//  ICASwiftApp
//
//  Created by Digital Trust Tech on 05/05/22.
//

import UIKit

class CustomTableViewCell: UITableViewCell {
    
    @IBOutlet var keyData: UILabel!
    @IBOutlet var valueData: UILabel!
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }

    override func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }

}
